src/views/Records/Record.vue

```diff
@@ -64,24 +64,54 @@
               <span>This record is hidden!</span>
             </v-alert>
 
-
-            <v-alert
-              v-if="ownershipApproved"
-              dense
-              type="success"
-              class="mb-1 flex-grow-1"
+            <transition
+              name="fade"
+              mode="in-out"
+              appear
             >
-              <span>Your request to maintain this record has been approved.</span>
-            </v-alert>
+              <div v-if="queryTriggered">
+                <div v-if="!queryTriggered">
+                  <v-alert
+                    v-if="(!alreadyClaimed) && ((!ownershipApproved && !alreadyClaimed) && (!alreadyNotClaimed && !error))"
```

Comment:
Although this PR appears to resolve the issue, I'm finding it hard to follow this if statement. One thing that is problematic for me is that I'm not sure of the difference between alreadyClaimed and alreadyNotClaimed. I can't find a definition of this in Record.vue.
Is there anything you could add which would clarify what this does, for the sake of future maintenance?

Reply:
please pull and check the comments added to clarify your concern. tnx

Reply:
Thanks for the comments. I'm afraid it's still somewhat confusing. For example, I'm not sure why there's a separate alreadyNotClaimed. Would this not be !alreadyClaimed, or is there a subtle difference I've not grasped?
The quoted code above is also rather hard to read and so it's difficult to tell what it does. Consider, for example, the long final v-if with various clauses, and the `if queryTriggered` followed immedately by an `if !queryTriggered` If you are able to refactor it to simply these statements it would help.
If not then something along the lines of:

```
<!-- At this point such and such a thing is happening -->
<div v-if="queryTriggered">
  <!-- Here, queryTriggered is checked again because... -->
     <div v-if="!queryTriggered">
     <!-- This component should only be shown if [explanation] -->
```

Reply:
yes I understand, the code is complex because a banner can have 5 status and some of the banners loads before loading the page:
1- error before loading (queryTriggered=false)
2- error after loading (queryTriggered=true)
3- approve 
4- reject
5- user only claimed ownership so the orange banner will be shown.

Reply:
Indeed, it does seem rather complex. 
So, some comments in the code to let the reader know what's going on before each section would be useful. Particularly a description of `(!alreadyClaimed) && ((!ownershipApproved && !alreadyClaimed) && (!alreadyNotClaimed && !error))` (assuming it can't be simplified).

Reply:
@knirirr your request is done in my last commit. tnx

Reply:
I think that, in order to get rid of the bug in production, I'll have to approve this for now and write a separate ticket for the remaining matters. 
I've changed the name of one variable to make it a little clearer when it comes to that.